### DISCUSSION TITLE
.setActivity resetting after a few minutes

### DISCUSSION THREAD

**Discussion by RagnarLothbrok-Odin** [discordjs/discord.js#7717] [Q&A] [answered]
Can someone help me understand why my bots status keeps resetting to nothing?
The following is my code I am running, it is ran at the `ready` event and is the only code in my entire source that uses `.setActivity`
The status works initally and usually resets to nothing after around 3 minutes.
```js
        this.client.user.setActivity(`test`, { type: 'WATCHING' });
```
This has been annoying and I have been trying to throw everything at it, I have tried the following:
`debug, rateLimit, warn, error` and `unhandledRejection`
I received no output from any of these when the status gets reset.

**Comment by Smidul** [comment]
That's an issue from the Discord side, djs can't do anything about it

**Comment by RagnarLothbrok-Odin** [comment, marked as answer]
Found cause; issue with code in the same file causing errors with the .setActivity
